src/core/document/json.pl

```diff
@@ -586,6 +592,11 @@
     create_context(Askable, Query_Context),
     database_prefixes(Query_Context, Context).
 
+database_and_default_prefixes(Askable, Prefixes) :-
+    database_prefixes(Askable, Database_Prefixes),
+    default_prefixes(Default_Prefixes),
```

Comment:
if we're gonna have a set of default prefixes, we should probably enforce that those prefixes cannot be used by users in their own graphs. Do we do that?

Reply:
I don't think that is necessary. If someone uses a prefix to mean something else, and they submit something else, that's their problem not ours.